src/test/java/com/amazon/opendistroforelasticsearch/security/test/DynamicSecurityConfig.java

```diff
@@ -50,6 +50,7 @@
     private String securityInternalUsers = "internal_users.yml";
     private String securityActionGroups = "action_groups.yml";
     private String securityNodesDn = "nodes_dn.yml";
+    private String securityAudit = "audit.yml";
```

Comment:
nit: static

Reply:
Class usage is nit static and this member is modified by setSecurityAudit() function defined below